src/pages/ProvideLiquidity/index.tsx

```diff
@@ -68,30 +86,42 @@ export default function ProvideLiquidity() {
     return <ChangeNetworkModal />
   }
 
+  const cardParams: { title: string; body: string; cta: string; url: string } = showLegacyAddLiquidity
```

Comment:
nit: Should we show *both* cards if we detect that the user already has liquidity, and maybe have this one in a  different color so it pops out a bit more? (not totally sure if showing both is worth, but might still be helpful to make this warning stand out a bit)

Reply:
fine with this if u wanna do it but I am trying to wrap up tbh